server/src/com/cloud/template/TemplateManagerImpl.java

```diff
@@ -275,6 +277,8 @@
     private StorageCacheManager cacheMgr;
     @Inject
     private EndPointSelector selector;
+    @Inject
+    private ImageStoreDao _imgStoreDao;
```

Comment:
hi @syed,
could you remove the "_" character from the variable name? the underscore is usually used in c++ programs to name private variables but it is not a convention in Java.
Ty.